src/realm/sync/network/default_socket.cpp

```diff
@@ -10,8 +10,8 @@ namespace {
 class DefaultWebSocketImpl final : public WebSocketInterface, public Config {
```

Comment:
unrelated to this PR, but I find `Config` name strange for a base class

Reply:
Definitely - it kinda makes things confusing. Luckily it's scoped to the `sync::websocket` namespace, but it is a set of abstract callback functions for the internal websocket implementation... 🤷‍♂️

Reply:
Very confusing. Let's not expand the scope of this, but can you file a like p5 trivial ticket to take care of this in a followup PR?